plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/AbstractDebugger.java

```diff
@@ -709,6 +709,15 @@ public void setValue(final Variable variable, final long threadId, final int fra
     }
   }
 
+  @Override
+  public Promise<? extends Location> getStackFrameLocation(long threadId, int frameIndex) {
+    if (isConnected()) {
+      return service.getStackFrameLocation(debugSessionDto.getId(), threadId, frameIndex);
+    }
+
+    return Promises.reject(JsPromiseError.create("Debugger is not connected"));
```

Comment:
If I'm not mistaken this method is deprecated and we should use org.eclipse.che.api.promises.client.PromiseProvider#reject(org.eclipse.che.api.promises.client.PromiseError), no?

Reply:
@dkuleshov make sense